core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -90,8 +87,12 @@ object ConsumerPerformance {
       consumerConnector.shutdown()
     }
     val elapsedSecs = (endMs - startMs) / 1000.0
-    if (!config.showDetailedStats) {
-      val totalMBRead = (totalBytesRead.get * 1.0) / (1024 * 1024)
+    val totalMBRead = (totalBytesRead.get * 1.0) / (1024 * 1024)
```

Comment:
Was it intentional to increase the scope of this val? It doesn't seem like it's used elsewhere.